tv.kodi.Kodi.yml

```diff
@@ -78,7 +79,34 @@ modules:
       - install -Dm644 tv.kodi.Kodi.metainfo.xml /app/share/metainfo/tv.kodi.Kodi.metainfo.xml
     config-opts:
       - -DVERBOSE=1
+      - -DUSE_LTO=${NUMCPU}
+      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+      - -DCMAKE_INSTALL_PREFIX=/usr
       - -DENABLE_TESTING=OFF
+      - -DENABLE_AIRTUNES=ON
+      - -DENABLE_ALSA=ON
+      - -DENABLE_AVAHI=ON
+      - -DENABLE_BLURAY=ON
+      - -DENABLE_CEC=ON
+      - -DENABLE_DBUS=ON
+      - -DENABLE_DVDCSS=ON
+      - -DENABLE_EGL=ON
+      - -DENABLE_EVENTCLIENTS=ON
+      - -DENABLE_INTERNAL_DAV1D=ON
+      - -DENABLE_INTERNAL_FFMPEG=ON
+      - -DENABLE_INTERNAL_CROSSGUID=OFF
```

Comment:
sorry if I wasn't clear about this, those 3 don't need to be internal, if the deps exist.

Reply:
Yes, of course. Will edit!